source/sourcing_and_processing_data/harvesters/dbnomics.rst

```diff
@@ -0,0 +1,37 @@
+DB.nomics harvester
+=========================
+
+Retrieve one or more datasets from `DB.nomics <https://db.nomics.world/>`_.
+
+A dataset is an aggregation of time series. Using the provider name and the dataset ID, it is possible to either fetch each of these time series individually as one OpenDataSoft dataset or keep them aggregated in a single OpenDataSoft dataset.
+
+A dataset ID follows the following format *ProviderName-DatasetId*.
+
+.. admonition:: Retrieving the dataset ID
+   :class: important
+
+   Retrieve the dataset id using the `explorer <https://db.nomics.world/views/explorer>`_.
+   Select a provider, the dataset to retrieve and click on API Link > JSON.
+   A link following this format will appear :
+   https://api.db.nomics.world/api/v1/json/datasets/bea-fa2004-section1-101-a/values
```

Comment:
For this block, you should either add dashes in front of each item to build a list, or add blank lines between items so they will appear in different paragraphs. 

Here, the compilation will produce one paragraph with no line breaks between items.

<img width="701" alt="screen shot 2018-03-28 at 11 44 08" src="https://user-images.githubusercontent.com/2072487/38021519-5c0a5e1a-327d-11e8-8291-07201ba8ab8d.png">

Reply:
Actually this one was more of readability in the .rst file, but it could be better to do it for the output too